src/components/FilterBar/FilterDisplay/FilterInput.react.js

```diff
@@ -6,14 +6,17 @@ export class FilterInput extends React.Component {
   }
 
   onClick() {
-    this.context.filterBarActor.disableFilter(this.props.filterUid);
+    const { groupKey, inputKey } = this.props;
+    this.context.filterBarActor.disableFilter(groupKey, inputKey);
```

Comment:
Well, I think it's a bad naming here. New logic allows you to use filter as many times as you want in one or many groups. Potentially method should be called `removeFilterValue` or `unsetFilterValue` & etc because in current configuration is nothing to disable.

Reply:
Ah, right. Yeah, I think remove or unset would be a more descriptive method name then.